### PR TITLE
New donors and SOP files

### PR DESCRIPTION
**Pull request by nsamanta**
1. Added docs for four new donors 138, 140, 144 & 145. 
2. Also adding relevant files for new CITE-seq SOP

### CODE REVIEW

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 1]
```diff
@@ -0,0 +1,1869 @@
...
+            margin-right: auto;
+        }
+    </style>
+</head>
+
+<body>
+    <img src="/assets/workflow/Islet Physiology Studies/HPAP-logo.jpg"
+        alt="perifusion" width="200" height="90"
+        style="vertical-align:middle" />
+    </br>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:17]: line break is ```<br>```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 2]
```diff
@@ -0,0 +1,1869 @@
...
+            Derived Tags and Hashtag Oligos</strong>
+    </H3>
+    <p align="center">Compiled by Vincent Wu and Gregory Golden of Betts
+        Lab<br />
+        Date submitted: May 2023</p>
+
+    <ol type="I">
+        <li>
+            <font size="+1"><strong>Description</strong></font>
+        </li></br>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:29]: line break is ```<br>```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 3]
```diff
@@ -0,0 +1,1869 @@
...
+            <td width="428" nowrap="nowrap" valign="middle">
+                <p align="center">DNase I&#44; 10 U&#47;&#956;L in glycerol</p>
+            </td>
+            <td width="119" nowrap="nowrap" valign="middle">
+                <p align="center">Roche Diagnostics</p>
+            </td>
+            <td width="109" nowrap="nowrap" valign="middle">
+                <p align="center">4716728001</p>
+            </td>
+            </tr>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:149]: missing opening tag for this ```</tr>```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 4]
```diff
@@ -0,0 +1,1869 @@
...
+            <td width="428" nowrap="nowrap" valign="middle">
+                <p align="center">1&#46;5ml DNA LoBind Tubes</p>
+            </td>
+            <td width="119" nowrap="nowrap" valign="middle">
+                <p align="center">Eppendorf</p>
+            </td>
+            <td width="109" nowrap="nowrap" valign="middle">
+                <p align="center">022431021</p>
+            </td>
+            </tr>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:230]: missing opening tag for this ```</tr>```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 5]
```diff
@@ -0,0 +1,1869 @@
...
+            <td width="428" nowrap="nowrap" valign="middle">
+                <p align="center">Dual Index Kit TT Set A&#44; 96 rxns</p>
+            </td>
+            <td width="119" nowrap="nowrap" valign="middle">
+                <p align="center">10X Genomics</p>
+            </td>
+            <td width="109" nowrap="nowrap" valign="middle">
+                <p align="center">1000215</p>
+            </td>
+            </tr>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:336]: missing opening tag for this ```</tr>```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 6]
```diff
@@ -0,0 +1,1869 @@
...
+
+                <p align="center"><b>cDNA Sample Index PCR amplification cycles
+                        done&#58; &#95;&#95;&#95;
+                        &#95;&#95;&#95; &#40;11 recommended&#41;</b></p>
+            </ol>
+        </ol>
+
+</body>
+
+</html>
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:1869]: extra empty line an the end of the file

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 7]
```diff
@@ -0,0 +1,1869 @@
...
+    </br>
+    <H3 align="center"><strong>HPAP CITEseq: Dual-index 3&#39; HT scRNAseq with
+            Antibody
+            Derived Tags and Hashtag Oligos</strong>
+    </H3>
+    <p align="center">Compiled by Vincent Wu and Gregory Golden of Betts
+        Lab<br />
+        Date submitted: May 2023</p>
+
+    <ol type="I">
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:26]: This ```<ol>``` does not have closing tag

File: assets/workflow/Immunobiology/CITEseq-protocol.html (hunk 1 above)
Comment (nsamanta) [reply assets/workflow/Immunobiology/CITEseq-protocol.html:17]: @yimsheng Thanks I see the error in the output terminal of Prettier on VScode now. I guess I was using a regular Terminal before. Thanks! I will resolve these issues.

**Comment by nsamanta** [comment]
@yimsheng Just made a new comment after resolving your comments. Thanks!

**Comment by nsamanta** [comment]
@dongbohu Yimin reviewed the HTML file. We can hopefully merge now

**Comment by nsamanta** [comment]
> https://github.com/faryabiLab/hpap-hub/blob/master/docs/coding_styles.md#41-the-following-rules-are-especially-important Change the indentation of your HTML file from 4 to 2!

Will do. That's strange, I had that in my HTML format settings which she helped me set up too.

**Comment by yimsheng** [comment]
> > https://github.com/faryabiLab/hpap-hub/blob/master/docs/coding_styles.md#41-the-following-rules-are-especially-important Change the indentation of your HTML file from 4 to 2!
> 
> Will do. That's strange, I had that in my HTML format settings which she helped me set up too.


@nsamanta 
Please check this setting under vscode->extension->prettier.

image

Formatting configuration can be double-checked in the output window.

image

**Comment by nsamanta** [comment]
> > > https://github.com/faryabiLab/hpap-hub/blob/master/docs/coding_styles.md#41-the-following-rules-are-especially-important Change the indentation of your HTML file from 4 to 2!
> > 
> > 
> > Will do. That's strange, I had that in my HTML format settings which she helped me set up too.
> 
> @nsamanta Please check this setting under vscode->extension->prettier tab width should be 2 though useTabs is false. Hope that works.
> 
> image
> Formatting configuration can be double-checked in the output window.
> 
> image
> Also please check the editor setting(you can search tab for this): image Editor is used when you write the code. Formattor is used when you format the code(right click and format the file with...).

@yimsheng Thanks! Yup, it's 2. Which is why I am confused why it changes to 4 when pushed to GitHub repo. But I re-formatted and committing the updated file now. Fingers crossed!

**Comment by nsamanta** [comment]
@yimsheng Can you check my latest commit, please? It looks like the indentation has been fixed. Thanks again. https://github.com/faryabiLab/hpap-public/pull/21/commits

**Comment by nsamanta** [comment]
> @yimsheng Can you check my latest commit, please? It looks like the indentation has been fixed. Thanks again. https://github.com/faryabiLab/hpap-public/pull/21/commits

Here is a copy of my terminal showing `tabwidth`:
```shell
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
```

File: assets/workflow/Immunobiology/CITEseq-protocol.html [hunk 8]
```diff
@@ -0,0 +1,2232 @@
...
+            <td width="119" nowrap="nowrap" valign="middle">
+                <p align="center">Roche Diagnostics</p>
+            </td>
+            <td width="109" nowrap="nowrap" valign="middle">
+                <p align="center">4716728001</p>
+            </td>
+            </tr>
+        </table>
+        <br />
+>>>>>>> 5c48885ae588f67f1e8f958067ec0cacaa48628b
```
Comment (yimsheng) [inline assets/workflow/Immunobiology/CITEseq-protocol.html:254]: line 88-254 resolve merge confilcts

**Review by yimsheng — COMMENTED** [review]
please do not leave unresolved merge conflicts in code
image

**Comment by yimsheng** [comment]
@nsamanta 
Would you mind formatting the code with formatter again? The style looks weired this time.

**Comment by nsamanta** [comment]
> @nsamanta Would you mind formatting the code with formatter again? The style looks weired this time.

@yimsheng Done. Just pushed the new commit

**Comment by yimsheng** [comment]
> > @nsamanta Would you mind formatting the code with formatter again? The style looks weired this time.
> 
> @yimsheng Done. Just pushed the new commit

@nsamanta The code looks good. Sorry about the back and forth. Once the settings and configs are done it would be much easier in the future.

**Comment by nsamanta** [comment]
> > > @nsamanta Would you mind formatting the code with formatter again? The style looks weired this time.
> > 
> > 
> > @yimsheng Done. Just pushed the new commit
> 
> @nsamanta The code looks good. Sorry about the back and forth. Once the settings and configs are done it would be much easier in the future.

THANK YOU!

**Comment by nsamanta** [comment]
@dongbohu I thnk we are finally ready to merge!

**Comment by dongbohu** [comment]
Great! I am merging and closing this PR.

**Pull request merged**